src/components/Header/MobileHeader.js

```diff
@@ -272,49 +272,54 @@ class MobileHeader extends React.Component {
   renderMyNyplButton() {
     let myNyplClass = '';
     const gaAction = (this.props.patronName) ? 'MyAccount' : 'LogIn';
-    let icon = <LoginIcon className="MobileMyNypl LoginIcon" />;
+    let icon = <LoginIcon className="LoginIcon" ariaHidden />;
     if (this.props.patronName) {
-      icon = <LoginIconSolid className="MobileMyNypl LoginIcon-loggedIn animated fadeIn" />;
+      icon = <LoginIconSolid className="LoginIcon-loggedIn animated fadeIn" ariaHidden />;
     }
     let buttonStyles = styles.inactiveMyNyplButton;
-    let buttonLabel = 'Open Log In Dialog';
-    let dialogWindow = null;
+    let buttonLabel = (this.props.patronName) ? 'My Account' : 'Login';
```

Comment:
I know ticket says otherwise, but shouldn't this be "Log In"

Reply:
Yes, I think you're right. @seanredmond  will be making this "Log In" instead of "Login".